### PR TITLE
52 objectives change request

### PR DESCRIPTION
**Pull request by nymann**
I am struggling to write anything relevant here.

### CODE REVIEW

**Review by Kechr — APPROVED** [review]
Looks good

File: project_proposal/project_proposal.md [hunk 1]
```diff
@@ -43,13 +29,17 @@ front-end to receive data via fetch calls in JavaScript it needs to talk with
...
 The group wants to host the REST API and Website on a Linux based server on a
 fully-qualified domain name using the container technology Docker which should
 be configured via Docker Compose. Furthermore the group aims to setup a
-continuous integration pipeline for development and continuous delivery via 
-Watchtower.
+continuous integration pipeline for development via GitHub Actions and
+continuous delivery via Watchtower.
 
-#### Initial Requirement Analysis
+### Initial Requirement Analysis
```
Comment (kkmp-dk) [inline project_proposal/project_proposal.md:42]: I am not sure to write here?

Comment (Kechr) [reply project_proposal/project_proposal.md:42]: The green one = ###

Comment (kkmp-dk) [reply project_proposal/project_proposal.md:42]: Not sure what you mean?

Comment (Kechr) [reply project_proposal/project_proposal.md:42]: Wait, why are we on this pull request @kepet19 ?

Comment (kkmp-dk) [reply project_proposal/project_proposal.md:42]: We are reviewers and should suggest what to write here?

Comment (kkmp-dk) [reply project_proposal/project_proposal.md:42]: Initial Requirement Analysis:
- There should be and API to talk to the beer machine
- Front end that is writing in JavaScript and display with HTML and CSS.
- Database to hold data about batches. 
- Calculated the optimal speed, the beer machine should run at. the speed should be right to get maximum of products.

Comment (nymann) [reply project_proposal/project_proposal.md:42]: huh? I just corrected the formatting, it's not my section

Comment (kkmp-dk) [reply project_proposal/project_proposal.md:42]: Sorry did not see that. I merged it already.

**Pull request merged**